apps/editing-toolkit/editing-toolkit-plugin/wpcom-block-editor-nux/src/block-editor-nux.js

```diff
@@ -1,6 +1,3 @@
-/*** THIS MUST BE THE FIRST THING EVALUATED IN THIS SCRIPT *****/
-import './public-path';
-
```

Comment:
@alshakero Not sure if you know the answer to this, but do you know if there's any reason to keep this code?